src/modules/filelist/icons/FileThumbnail.tsx

```diff
@@ -90,14 +93,22 @@ const FileThumbnail: React.FC<FileThumbnailProps> = ({
           className="u-flex u-flex-items-center u-flex-justify-center u-bdrs-4"
           width={size}
           height={size}
-          bgcolor="var(--contrastBackgroundColor)"
+          bgcolor={viewType === 'list' ? 'var(--contrastBackgroundColor)' : ''}
         >
           {isKonnectorFolder ? (
             <BadgeKonnector file={file}>
-              <FileIcon file={file} size={48} isEncrypted={isEncrypted} />
+              <FileIcon
+                file={file}
+                size={viewType === 'grid' ? 96 : 48}
```

Comment:
why don't you pass `96` when you call `FileThumbnail`? Like that, no need to rely on the hook.